deploy/cluster_role.yaml

```diff
@@ -15,12 +15,13 @@ rules:
   resources:
   - '*'
   verbs:
-  - 'get'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'delete'
-  - 'watch'
+  - create
```

Comment:
This change is mainly to reduce the diff between new operators and this operator, in an attempt to make it easier to compare what changed between SDK versions.

The new list contains `patch`, which the old list didn't. Apart from that, it's only ordering and formatting changes.